### DISCUSSION TITLE
[English] - I'm an Australian and want to help

### DISCUSSION THREAD

**Discussion by MrDwarf7** [rbatis/rbatis#565] [General]
# Hello!

### I'm an Aussie that has the (unfortunate) circumstance of dealing with MSSQL on a regular basis, and by extension - the rather crappy ecosystem for MSSQL support for Rust.

This crate gives me hope that weird errors and tons of boiler-plate could (maybe) become a distant memory.
I love the idea as a whole and want to assist in making the documentation clearer, especially for English speakers.

Not sure if I'd be able to contribute all that much to the actual Rust code, but documentation?
Yeah, I can absolutely I can help with that!

So that then begs the question - how do I go about actually contributing to documentation (IE: the website etc.)?
If I need clarification on something from the author that isn't clear between either the docs or the code itself, what's the process?

## [**Note**]  
For any other English speakers (Or other Aussie's!) lets help 'ol mate out here and make it a solid option in the ecosystem 💪

**Comment by zhuxiujia** [comment]
You can point out or submit the unreasonable or unclear parts in the document,  just send any PR for [doc_website](https://github.com/rbatis/rbatis.io)

**Reply by MrDwarf7** [reply]
Discussion on content and how you would like to convey it to be done via the PR, correct?
